src/registrar/templates/domain_detail.html

```diff
@@ -5,6 +5,28 @@
   {{ block.super }}
   <div class="margin-top-4 tablet:grid-col-10">
 
+    <div
+        class="usa-summary-box dotgov-status-box margin-top-3 padding-left-2{% if domain.state == 'unknown' or domain.state == 'dns needed'%} dotgov-status-box--action-need{% endif %}" 
+        role="region"
+        aria-labelledby="summary-box-key-information"
+    >
+      <div class="usa-summary-box__body">
+        <p class="usa-summary-box__heading font-sans-md margin-bottom-0" 
+          id="summary-box-key-information"
+        > 
+          <span class="text-bold text-primary-darker">
+            Status:
+          </span>
+            {% if domain.state == "unknown" or domain.state == 'dns needed'%}
```

Comment:
```suggestion
            {% if domain.state == domain.State.UNKNOWN or domain.state == domain.State.DNS_NEEDED%}
```
(Optional) Same thing here, just helps with maintainability a little